src/constants.h

```diff
@@ -172,7 +172,7 @@ CONSTEXPR(uint32_t, releaseMonitorMsec, 21600000, 4000, 0)
 
 // in milliseconds, how often we should fetch the server list, the account and
 // so on.
-CONSTEXPR(uint32_t, schedulePeriodicTaskTimerMsec, 3600000, 30000, 0)
+CONSTEXPR(uint32_t, schedulePeriodicTaskTimerMsec, 3600000, 300000, 0)
```

Comment:
Just because arithmetic is hard, please add a comment here with this time in hours or minutes.

Reply:
Is 5 minutes too long in debug for testing other scenarios? The PR description says that the previous 30 seconds made testing difficult. Could you please clarify why?